src/components/BeamContext.tsx

```diff
@@ -8,7 +8,7 @@ import { EmptyRef } from "src/utils/index";
 /** The internal state of our Beam context; see useModal and useSuperDrawer for the public APIs. */
 export interface BeamContextState {
   // SuperDrawer contentStack
-  contentStack: MutableRefObject<ContentStack[]>;
+  contentStack: MutableRefObject<readonly ContentStack[]>;
```

Comment:
**Question:** This didn't seem to help with the `.push` - does it help with other things?

Reply:
It did cause compile errors for me when I directly to put the `push` calls back, :shrug: